client/src/components/campaign/round/round.html

```diff
@@ -0,0 +1,135 @@
+<div class="round"
+      layout="row" layout-align="center start"
+      ng-class="{ 'round--last': $ctrl.last }">
+  <div class="round__status {{'round__status--' + $ctrl.round.status}}"
+        layout="row" layout-align="center center">
+    <span ng-if="$ctrl.round.vote_method === 'yesno'"><md-icon>thumbs_up_down</md-icon></span>
+    <span ng-if="$ctrl.round.vote_method === 'rating'"><md-icon>star_border</md-icon></span>
+    <span ng-if="$ctrl.round.vote_method === 'ranking'"><md-icon>sort</md-icon></span>
+  </div>
+  <span class="round__line"></span>
+  <div flex>
+    <h2 class="round__name"
+        layout="row" layout-align="space-between center">
+      <div layout="column" layout-align="center start">
+        <span>{{ ::$ctrl.round.name }}</span>
+        <span class="muted">{{ ::$ctrl.round.vote_method }} · {{ $ctrl.round.status }}</span>
+      </div>
+      <md-button
+          ng-click="$ctrl.editRound()"
+          ng-disabled="$ctrl.round.status !== 'paused'">
```

Comment:
Would be useful to have a tool tip or something to point out that a round needs to be paused to edit.